packages/docs/configuration/readme.md

```diff
@@ -186,32 +149,23 @@ module.exports = {
         l: 3,
         xxl: 4,
       },
+      prefix: 'legacy-',
```

Comment:
Not convinced by the `legacy-` word : i'd find it awkward in a running project.

Why not be more explicit with `float-grid-*` or `f-grid-*` for instance ?

Reply:
I like the `float-grid-*` which describes exactly what this grid system does.

Reply:
Ok @notjb @titouanmathis Let's go for `float-grid-*` 🙂